src/navigation-bar-customization/add-delete-link/applications.json

```
@@ -41,28 +44,14 @@
         "title": "Sample Link 3",
         "description": "This sample shows how to add a new link to More menu",
         "payload": {
-          "link": "http://www.ibm.com",
+          "link": "http://www.hcl.com",
```

```suggestion
          "link": "https://www.hcl.com",
```
In other places, we use hcltechsw as the example. Should we use a consistent site across examples?

---

will use www.hcl-software.com